network/network.go

```diff
@@ -0,0 +1,122 @@
+package network
+
+import (
+	"context"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/network"
+	"github.com/google/uuid"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// New creates a new network with a random UUID name, calling the already existing GenericNetwork APIs.
+// Those existing APIs are deprecated and will be removed in the future, so this function will
+// implement the new network APIs when they will be available.
+// By default, the network is created with the following options:
+// - Driver: bridge
+// - Attachable: true
+// - Labels: the Testcontainers for Go generic labels, to be managed by Ryuk. Please see the GenericLabels() function
+// And those options can be modified by the user, using the CreateModifier function field.
+func New(ctx context.Context, opts ...NetworkCustomizer) (*testcontainers.DockerNetwork, error) {
+	nc := types.NetworkCreate{
+		Driver:     "bridge",
+		Attachable: true,
```

Comment:
Can you elaborate more about the having it with `true` value, please? Looking at the docs and the java implementation, seems like is not needed.

Docs

> Globally scoped network is manually attachable by regular containers from workers in swarm mode.

Reply:
Mmmm I missed the official docs connect this option with Docker swarm. You're totally right and it's not needed it. Removing it